assets/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "license": "Apache-2.0",
   "devDependencies": {
+    "@tailwindcss/custom-forms": "^0.2.1",
```

Comment:
This package is deprecated in favor of `@tailwindcss/forms`. See https://github.com/tailwindlabs/tailwindcss-custom-forms#readme.

Reply:
Thank you, working on it